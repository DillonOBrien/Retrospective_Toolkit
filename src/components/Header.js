/**
 * Stateless component that displays the header for all of the landing pages.
 */


import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Button from '@material-ui/core/Button';
import Typography from '@material-ui/core/Typography';
import { Toolbar } from '@material-ui/core';
import logo from '../resources/statefarmLogo.svg';

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
    margin: '10px',
    paddingBottom: '10px',
  },
  title: {
    flexGrow: 1,
    display: 'flex',
    justifyContent: 'flex-end',
    color: 'red',
    fontWeight: "bold",
    marginRight: theme.spacing(2),
  },

  toolbar: {
    color: 'black',
    background: 'whitesmoke',
  },
  menuButton: {
  },
  image: {
    height: '6vmin',
    paddingBottom: '1vmin',
  },
}));

const Header = () => {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <AppBar position="static" className={classes.toolbar}>
        <Toolbar>
          <img src={logo} className={classes.image} alt="logo" />
          <Typography variant ="h5" className={classes.title}>
            RETROSPECTIVE TOOLKIT
          </Typography>
          <Typography variant="span" className={classes.title} >
            Developed by High Rollers
          </Typography>
          <Button color="inherit">Login</Button>
        </Toolbar>
      </AppBar>
    </div>
  )

}

export default Header;