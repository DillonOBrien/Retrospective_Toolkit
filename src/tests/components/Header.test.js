import React from "react";
import { shallow } from "enzyme";
import Header from "../../components/Header";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Header />);
});

describe("Components/Header Testing", () => {
  // Snapshot for header
  test("SNAPSHOT: Header", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
