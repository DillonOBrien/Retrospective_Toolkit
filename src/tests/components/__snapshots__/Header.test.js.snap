// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Header Testing SNAPSHOT: Header 1`] = `
<Fragment>
  <HideOnScroll>
    <WithStyles(ForwardRef(AppBar))
      className="makeStyles-toolbar-2"
    >
      <WithStyles(ForwardRef(Toolbar))
        className="makeStyles-root-1"
      >
        <Link
          to="/"
        >
          <img
            alt="logo"
            src="https://d1g31diwtzkeb3.cloudfront.net/statefarmLogo.svg"
          />
        </Link>
        <div
          className="makeStyles-grow-3"
        />
        <Link
          className="makeStyles-menuButton-4"
          to="/login"
        >
          <WithStyles(ForwardRef(Button))>
            <Memo([object Object])
              fontSize="large"
            />
            Login
          </WithStyles(ForwardRef(Button))>
        </Link>
      </WithStyles(ForwardRef(Toolbar))>
    </WithStyles(ForwardRef(AppBar))>
  </HideOnScroll>
  <WithStyles(ForwardRef(Toolbar)) />
</Fragment>
`;
